src/formElementPlugins/listBuilder/src/ApplicationView.tsx

```diff
@@ -243,10 +243,10 @@ const buildElements = async (
     category: field.category as TemplateElementCategory,
     title: field.title as string,
     parameters: field.parameters,
-    validationExpression: field?.validation || true,
+    validationExpression: !!field?.validation,
     validationMessage: field?.validationMessage || '',
-    isVisibleExpression: field?.visibilityCondition || true,
-    isEditableExpression: field?.isEditable || true,
+    isVisibleExpression: !!field?.visibilityCondition,
+    isEditableExpression: !!field?.isEditable,
```

Comment:
Just wondering -- what's the advantage of this way of expressing it?

Reply:
I am not sure why I did that, it's only good for booleans, those are expressions, fixed:
63f304446b6a7204b5549c49087ab7e8ec5c654a
it also fixes
> The validation expression is not showing up in the GUI. In "Common Properties", there is an "isValid" field, but it's showing as "true" no matter what, even if there should be an expression there.